code/modules/mob/living/simple_animal/hostile/mining_mobs/hivelord.dm

```diff
@@ -349,7 +349,7 @@
 			mask = /obj/item/clothing/mask/breath
 		if("Operative")
 			id_job = "Operative"
-			outfit = /datum/outfit/syndicatecommandocorpse
+			outfit = /datum/outfit/syndicateminercorpse //yogs - no more 8 TC hardsuit for killing a lavaland mob
```

Comment:
Is that an unmentioned change? Excuse me what the fuck!?

Reply:
No, this is mentioned in the changelog. Lowest tweak.